desktop.css

```diff
@@ -0,0 +1,375 @@
+@media only screen and (min-width: 768px) {
+  @import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+
+  * {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+  }
+
+  body,
+  html {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+  }
+
+  main {
+    width: 100%;
+    height: 4561px;
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .desktop-navbar-container {
+    display: block;
+    width: 100%;
+    height: 72px;
+    background-color: #3e3c3c;
+  }
+
+  .desktop-navbar {
+    display: flex;
+    gap: 8px;
+    padding: 1.5% 18% 0 10%;
+  }
```

Comment:
[OPTIONAL] Kindly make sure your navigation is aligned center and the elements between the nav link has same space between each other like on the Figma template.
<img width="1440" alt="Screenshot 2023-03-02 at 9 43 09 PM" src="https://user-images.githubusercontent.com/25887253/222522989-afbaf763-3e95-4988-9f1a-5248962e84dc.png">
<img width="387" alt="Screenshot 2023-03-02 at 9 45 19 PM" src="https://user-images.githubusercontent.com/25887253/222522991-71573924-7448-4c24-9125-5e93dc733b87.png">